### PR TITLE
added unit and integration test badges to README

### PR DESCRIPTION
**Pull request by engelmi**
Added unit and integration test badges to README. 

Preview:
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 # BlueChi: A systemd service controller for multi-node environments
 
 [![Copr build status](https://copr.fedorainfracloud.org/coprs/g/centos-automotive-sig/bluechi-snapshot/package/bluechi/status_image/last_build.png)](https://copr.fedorainfracloud.org/coprs/g/centos-automotive-sig/bluechi-snapshot/package/bluechi/)
+
```
Comment (mwperina) [inline README.md]: Please remove this empty line, it would be better to have all badges on the same line

Comment (engelmi) [reply README.md]: I wanted to group the test-related badges and separate the copr build one. I can remove the line, though.

Comment (engelmi) [reply README.md]: Changed.

**Review by mwperina — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (mwperina) [reply README.md]: :+1:  Thanks

**Pull request merged**